network/openvswitch_port.py

```diff
@@ -17,6 +19,11 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this software.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Portions copyright @ 2015 VMware, Inc.  All rights reserved.
```

Comment:
keep copyright with existing above, also not sure 'All rights reserved' works with GPL code